Env issue found in perf test - resource group could not be found

## In what area(s)?

/area test-and-release

## What version of Dapr?

edge: output of `git describe --dirty`

## Expected Behavior

The perf test should run with correct azure resource group.

```bash
Run az aks get-credentials -n "***39dad65d86-aks" -g "Dapr-Perf-***39dad65d86"
  az aks get-credentials -n "***39dad65d86-aks" -g "Dapr-Perf-***39dad65d86"
  kubectl create namespace dapr-tests
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    GOVER: 1.18
    GOPROXY: https://proxy.golang.org
    KUBECTLVER: v1.[2](https://github.com/dapr/dapr/runs/6894281600?check_suite_focus=true#step:13:2)2.6
    HELMVER: v[3](https://github.com/dapr/dapr/runs/6894281600?check_suite_focus=true#step:13:3).7.2
    DAPR_NAMESPACE: dapr-tests
    MAX_TEST_TIMEOUT: 5[4](https://github.com/dapr/dapr/runs/6894281600?check_suite_focus=true#step:13:4)00
    AZURE_REGIONS: westus3
    DAPR_CACHE_REGISTRY: dapre2eacr.azurecr.io
    GOOS: linux
    GOARCH: amd64
    AZURE_STORAGE_ACCOUNT: ***
    AZURE_STORAGE_ACCESS_KEY: ***
    TEST_OUTPUT_FILE_PREFIX: /home/runner/work/dapr/dapr/test_report
    DAPR_CONTAINER_LOG_PATH: /home/runner/work/dapr/dapr/container_logs/perf_tests
    DAPR_TEST_LOG_PATH: /home/runner/work/dapr/dapr/test_logs/perf_tests
    CHECKOUT_REPO: dapr/dapr
    CHECKOUT_REF: refs/heads/master
    AZURE_HTTP_USER_AGENT: 
    AZUREPS_HOST_ENVIRONMENT: 
    TEST_PREFIX: ***39dad6[5](https://github.com/dapr/dapr/runs/6894281600?check_suite_focus=true#step:13:5)d8[6](https://github.com/dapr/dapr/runs/6894281600?check_suite_focus=true#step:13:6)
    DAPR_REGISTRY: ***39dad65d[8](https://github.com/dapr/dapr/runs/6894281600?check_suite_focus=true#step:13:8)6acr.azurecr.io
    TEST_CLUSTER: ***3[9](https://github.com/dapr/dapr/runs/6894281600?check_suite_focus=true#step:13:9)dad65d86-aks
    DAPR_TAG: ***39dad65d86
    DAPR_TEST_TAG: ***39dad65d86
    TEST_RESOURCE_GROUP: Dapr-Perf-***39dad65d86
ERROR: (ResourceGroupNotFound) Resource group 'Dapr-Perf-***39dad65d86' could not be found.
Code: ResourceGroupNotFound
Message: Resource group 'Dapr-Perf-***39dad65d86' could not be found.
Error: Process completed with exit code 3.
```

## Actual Behavior

Something the perf test case failed for resource group not found.

## Steps to Reproduce the Problem

This happens only on github action, I can list some failed run here. More information please check the log in the runs:

https://github.com/dapr/dapr/actions/runs/2495769687

https://github.com/dapr/dapr/actions/runs/2499416846

https://github.com/dapr/dapr/actions/runs/2499717828

https://github.com/dapr/dapr/actions/runs/2502044582

--- a/.github/workflows/dapr-perf.yml
+++ b/.github/workflows/dapr-perf.yml
@@ -90,8 +90,10 @@ jobs:
         if: env.CHECKOUT_REPO != ''
         run: |
           BASE_STR="PERF|${GITHUB_SHA}|${GITHUB_SERVER_URL}|${GITHUB_REPOSITORY}|${GITHUB_RUN_ID}|${GITHUB_RUN_ATTEMPT}"
+          echo "Base string is ${BASE_STR}"
           SUFFIX=$(echo $BASE_STR | sha1sum | head -c 10)
-          TEST_PREFIX="daprperf${SUFFIX}"
+          echo "Suffix is ${SUFFIX}"
+          TEST_PREFIX="daprprf${SUFFIX}"
           echo "Test prefix is ${TEST_PREFIX}"
           echo "TEST_PREFIX=${TEST_PREFIX}" >> $GITHUB_ENV
           echo "TEST_RESOURCE_GROUP=Dapr-Perf-${TEST_PREFIX}" >> $GITHUB_ENV
@@ -196,8 +198,10 @@ jobs:
         if: env.CHECKOUT_REPO != ''
         run: |
           BASE_STR="PERF|${GITHUB_SHA}|${GITHUB_SERVER_URL}|${GITHUB_REPOSITORY}|${GITHUB_RUN_ID}|${GITHUB_RUN_ATTEMPT}"
+          echo "Base string is ${BASE_STR}"
           SUFFIX=$(echo $BASE_STR | sha1sum | head -c 10)
-          TEST_PREFIX="daprperf${SUFFIX}"
+          echo "Suffix is ${SUFFIX}"
+          TEST_PREFIX="daprprf${SUFFIX}"
           echo "Test prefix is ${TEST_PREFIX}"
           echo "TEST_PREFIX=${TEST_PREFIX}" >> $GITHUB_ENV
           echo "DAPR_REGISTRY=${TEST_PREFIX}acr.azurecr.io" >> $GITHUB_ENV
@@ -318,8 +322,10 @@ jobs:
         if: env.CHECKOUT_REPO != ''
         run: |
           BASE_STR="PERF|${GITHUB_SHA}|${GITHUB_SERVER_URL}|${GITHUB_REPOSITORY}|${GITHUB_RUN_ID}|${GITHUB_RUN_ATTEMPT}"
+          echo "Base string is ${BASE_STR}"
           SUFFIX=$(echo $BASE_STR | sha1sum | head -c 10)
-          TEST_PREFIX="daprperf${SUFFIX}"
+          echo "Suffix is ${SUFFIX}"
+          TEST_PREFIX="daprprf${SUFFIX}"
           echo "Test prefix is ${TEST_PREFIX}"
           echo "TEST_PREFIX=${TEST_PREFIX}" >> $GITHUB_ENV
           echo "DAPR_REGISTRY=${TEST_PREFIX}acr.azurecr.io" >> $GITHUB_ENV
@@ -482,8 +488,10 @@ jobs:
       - name: Build test prefix
         run: |
           BASE_STR="PERF|${GITHUB_SHA}|${GITHUB_SERVER_URL}|${GITHUB_REPOSITORY}|${GITHUB_RUN_ID}|${GITHUB_RUN_ATTEMPT}"
+          echo "Base string is ${BASE_STR}"
           SUFFIX=$(echo $BASE_STR | sha1sum | head -c 10)
-          TEST_PREFIX="daprperf${SUFFIX}"
+          echo "Suffix is ${SUFFIX}"
+          TEST_PREFIX="daprprf${SUFFIX}"
           echo "Test prefix is ${TEST_PREFIX}"
           echo "TEST_PREFIX=${TEST_PREFIX}" >> $GITHUB_ENV
         shell: bash

